scrapper/freebiesglobal.py

```diff
@@ -0,0 +1,33 @@
+from bs4 import BeautifulSoup
+import logging
+import time
+
+import fetch
+import util
+
+
+async def get(session, url="https://freebiesglobal.com/dealstore/udemy"):
+    link_card = set()
+    links_udemy = set()
+    start = time.time()
+    num_calls = 1
+
+    cont = await fetch.get_all(session, url)
+    if cont:
```

Comment:
This is doing the same thing as `discudemy.py`. Really these should be instead calling a helper function parameterized by url and css selectors. This will avoid copy/paste errors and oversights